covid19_sir/model/base.py

```diff
@@ -133,6 +138,8 @@ def __init__(self, **kwargs):
         self.params['extroversion_mean']=kwargs.get("extroversion_mean",0.5)
         self.params['extroversion_stdev']=kwargs.get("extroversion_stdev",0.3)
         self.params['min_behaviors_to_copy']=kwargs.get("min_behaviors_to_copy",3)
+        self.params['debug_each_n_cycles']=kwargs.get("debug_each_n_cycles",20)
```

Comment:
`debug_each_n_cycles` is not a simulation parameter so it isn't supposed to be here.